### ISSUE TITLE
Ask about reproducing the style synthesis results

### ISSUE THREAD

**Issue by rkomatsu2020** [clovaai/stargan-v2#143]
Hi, I have tried implementing StarGAN v2 following the codes. After training, I met the poor generated images. Below results are output by trained my Generator for animal to animal translation. From left to right, content images, targeted style images and style synthesised images are displayed. As you could see, the output images had the random color images without content and style features.
![epoch50_fakeStyle_result](https://user-images.githubusercontent.com/76080261/180672211-6604508a-187f-4594-9017-b84e9fc7d89a.png)

About the parameters, I set

- latent noise dim = 16
- batch size = 8
- Adam
- -------Generator : learning rate = 1e-4, beta1=0.0, beta2=0.99, weight_decay=1e-4
- -------Discriminator : learning rate = 1e-4, beta1=0.0, beta2=0.99, weight_decay=1e-4
- -------Mapping Encoder (Latent Noise Encoder) : learning rate = 1e-6, beta1=0.0, beta2=0.99, weight_decay=1e-4
- -------Style Encoder (Style Image Encoder) : learning rate = 1e-4, beta1=0.0, beta2=0.99, weight_decay=1e-4

Could you give me the advices?

**Comment by jdunkelheit** [comment]
I'm experiencing the exact same issue. Even after considerable training time, the model only outputs noise.

![noise example](https://user-images.githubusercontent.com/112777797/188284363-4dd4b922-1c0f-45d3-861a-a16f35e5f9db.png)
